### PR TITLE
No sql example

### PR DESCRIPTION
**Pull request by markburns**
I've been trying to debug why our app is still hitting the db and thought I would confirm that active_mocker itself doesn't hit the db during tests when using mocks.

https://blog.pivotal.io/pivotal-labs/labs/testing-strategies-rspec-nulldb-nosql

Unfortunately for us, we confirmed that we can't see a problem in active_mocker.

As a side effect though, we have this test. Which might be useful for the example app.
But would you maybe rather have it in the actual test suites?

### CODE REVIEW

File: example_rails_app/spec/models/user_spec.rb [hunk 1]
```diff
@@ -0,0 +1,24 @@
+# frozen_string_literal: true
```
Comment (markburns) [inline example_rails_app/spec/models/user_spec.rb]: Realised this PR is a bit hard to parse with the mock generation and hound dog comments.
The main gist of the PR is from this point down. `example_rails_app/spec/models/user_spec.rb` and `example_rails_app/spec/rails_helper.rb`

**Comment by zeisler** [comment]
It does not make sense to put this test in the example app, nor do I think this test makes a lot of sense given a understanding how active mocker works.
